mappings/net/minecraft/container/CartographyTableContainer.mapping

```diff
@@ -1,6 +1,8 @@
 CLASS ayg net/minecraft/container/CartographyTableContainer
 	FIELD c inventory Lags;
 	FIELD d context Layk;
+	FIELD e itemTakingInProgress Z
+	FIELD f resultInventorySlot Lazi;
```

Comment:
resultSlot

Reply:
I was just naming it that way because it used a slot in the field called `inventory` before, but I guess it's still very clear what it means.
It's just sometimes hard for me to draw the line between too ambigious and too verbose.